lib/backend/model/node.go

```diff
@@ -219,3 +226,54 @@ func (options OrchRefListOptions) defaultPathRoot() string {
 func (options OrchRefListOptions) KeyFromDefaultPath(path string) Key {
 	return OrchRefKey{Hostname: options.Hostname}
 }
+
+// The Felix Wireguard Key.
+type WireguardKey struct {
```

Comment:
Understand this might be a late change, but we do send the entire node object to Felix now. https://github.com/projectcalico/libcalico-go/blob/master/lib/backend/syncersv1/updateprocessors/felixnodeprocessor.go#L150-L157

The idea being that we should, for new data, just use the v3 objects and not introduce new v1 config structures.